api/desecapi/views.py

```diff
@@ -300,10 +300,9 @@ def get_queryset(self):
         return rrsets
 
     def get_objects(self, request, many):
-        if many:
+        if many and isinstance(request.data, list):
```

Comment:
Under which circumstances is this called with `many=True` without `request.data` being a list? My understanding was that that's how the `many` argument was determined...

Reply:
in the code following, we try to iterate over `request.data`. To avoid an exception there, the check is conducted again (in case this is called from somewhere else).